src/indexer/allo/v2/parsePoolMetadata.ts

```diff
@@ -0,0 +1,22 @@
+type ipfsGetFn = <T>(cid: string) => Promise<T | undefined>;
```

Comment:
```suggestion
type IpfsGetFn = <T>(cid: string) => Promise<T | undefined>;
```

types are title case

Reply:
not sure if the name of the file makes sense with the contents, maybe just `poolMetadata.ts`?

Reply:
@boudra  update